WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/OrderDetailsDataSourceTests.swift

```diff
@@ -70,7 +70,7 @@ final class OrderDetailsDataSourceTests: XCTestCase {
         XCTAssertNotNil(issueRefundRow)
     }
 
-    func test_refund_button_is_not_visible_when_the_order_is_refunded() throws {
+    func test_refund_button_is_not_visible_when_the_order_status_is_refunded() throws {
```

Comment:
nit: might be nice having a test case on partial refunds too, where the refund button is visible after a previous refund 😄

Reply:
Good idea! I added them to `OrderRefundsOptionsDeterminerTests`:

`func test_isAnythingToRefund_when_not_all_items_are_refunded_but_amount_is_returns_true()`